tst/inet/mqtt_client/main.c

```diff
@@ -253,7 +253,112 @@ static int test_publish(struct harness_t *harness_p)
     return (0);
 }
 
-static int test_subscribe(struct harness_t *harness_p)
+static int test_subscribe_qos0(struct harness_t *harness_p)
```

Comment:
Leave the old testcase ``test_subscribe()`` unmodified, and create new test cases called ``test_incoming_publish_qos0-2``, or somthing similar. Makes it clear that they test the incoming publish functionality (as opposed to subscribe functionality).